release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -136,6 +136,13 @@ The updated location and guide name reflect a more focused documentation experie
 [id="ocp-4-17-openshift-cli_{context}"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-17-openshift-cli-oc-mirror-kubevirt_{context}"]
+==== oc-mirror to include the HyperShift KubeVirt CoreOS container
+
+With this release, oc-mirror now includes the {op-system-first} image for the HyperShift KubeVirt provider when mirroring the {product-title} release payload.
+
+When using KubeVirt as the provider for HyperShift, the KubeVirt VMs, which act as nodes for hosted clusters, require an {op-system} image shipped as a container-disk image. To support disconnected environments, this image is now included in the mirroring process.
```

Comment:
I would just spell out VM since it's the first mention of the term in this section:
```suggestion
When using KubeVirt as the provider for HyperShift, the KubeVirt virtual machines, which act as nodes for hosted clusters, require an {op-system} image shipped as a container-disk image. To support disconnected environments, this image is now included in the mirroring process.
```